google/cloud/storage/transfer_manager.py

```diff
@@ -495,7 +702,226 @@ def download_many_to_path(
     return download_many(
         blob_file_pairs,
         download_kwargs=download_kwargs,
-        threads=threads,
         deadline=deadline,
         raise_exception=raise_exception,
+        worker_type=worker_type,
+        max_workers=max_workers,
     )
+
+
+def download_chunks_concurrently(
+    blob,
+    filename,
+    chunk_size=DEFAULT_CHUNK_SIZE,
+    download_kwargs=None,
+    deadline=None,
+    worker_type=PROCESS,
+    max_workers=8,
+):
+    """Download a single file in chunks, concurrently.
+
+    This function is a PREVIEW FEATURE: the API may change in a future version.
+
+    In some environments, using this feature with mutiple processes will result
+    in faster downloads of large files.
+
+    Using this feature with multiple threads is unlikely to improve download
+    performance under normal circumstances due to Python interpreter threading
+    behavior. The default is therefore to use processes instead of threads.
+
+    Checksumming (md5 or crc32c) is not supported for chunked operations. Any
+    `checksum` parameter passed in to download_kwargs will be ignored.
```

Comment:
Good call out. Where are we ignoring the `checksum` parameter?

Reply:
Explained this offline, but for posterity, checksum is not sent by the server for ranged reads, and chunked downloads are all ranged reads, so none of the responses include a checksum. As a result, the resumable media code silently ignores the checksum parameter.